Engine/General/Logging.cs

```diff
@@ -14,13 +14,22 @@ public static void Load()
         {
             File.Delete("log.txt");
 
-            _logger = new LoggerConfiguration()
+            var logger = new LoggerConfiguration()
```

Comment:
Used local variable just for readability, since Roslyn will change it to inline when compiling with Release.